recipe/build.sh

```diff
@@ -25,6 +25,9 @@ cmake -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
       -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
       -DBUILD_TESTS=${BUILD_TESTS} \
       -DBUILD_BENCHMARKS=${BUILD_BENCHMARKS} \
+      -DCPM_LOCAL_PACKAGES_ONLY=1 \
```

Comment:
The preferred style is `-DCPM_LOCAL_PACKAGES_ONLY=ON`

Reply:
```suggestion
      -DCPM_LOCAL_PACKAGES_ONLY=ON \
```

Reply:
Thanks!